### PR TITLE
Brought Block hash back to the block RPC output

### PR DESCRIPTION
**Pull request by frol**

### CODE REVIEW

File: chain/jsonrpc/tests/rpc_query.rs [hunk 1]
```diff
@@ -16,7 +16,18 @@ fn test_block() {
 
         let mut client = new_client(&format!("http://{}", addr));
         actix::spawn(client.block(0).then(|res| {
-            assert_eq!(res.unwrap().header.height, 0);
+            let res = res.unwrap();
+            assert_eq!(res.header.height, 0);
+            assert!(res.header.epoch_hash.0.len() > 0);
```
Comment (bowenwang1996) [inline chain/jsonrpc/tests/rpc_query.rs]: Why not assert the exact length?

Comment (ilblackdragon) [reply chain/jsonrpc/tests/rpc_query.rs]: even better, why not assert the value? this is genesis block, values going to be the same except timestamp.

Comment (ilblackdragon) [reply chain/jsonrpc/tests/rpc_query.rs]: I guess hash itself will be different due to timestamp as well.

@bowenwang1996 exact length though due to bs58 can be different depending on value.

Comment (bowenwang1996) [reply chain/jsonrpc/tests/rpc_query.rs]: Yes but we can decode and then assert the length

**Comment by nearmax** [comment]
@frol The working branch is `staging`, not `master`. We merge our PRs into `staging` and then once we cut a new version one person merges `staging` into `master`.

**Comment by frol** [comment]
@nearmax This was a special request from @ilblackdragon to merge it directly to master.

**Comment by frol** [comment]
@nearmax without this hotfix, Explorer will be in a broken state for all that time till staging gets released.

**Pull request merged**